sisl/physics/electron.py

```diff
@@ -1615,14 +1615,16 @@ def norm2(self, sum=True):
         S = self.Sk()
         return conj(self.state) * S.dot(self.state.T).T
 
-    def inner(self, right=None, diagonal=True, align=False):
-        r""" Return the inner product by :math:`\mathbf M_{ij} = \langle\psi_i|\psi'_j\rangle`
+    def inner(self, right=None, matrix=None, diagonal=True, align=False):
```

Comment:
is `matrix` too long, or should we just use `A`, or `M`?

Also, some places I use `diagonal` others `diag`, which do you prefer? Numpy has the `np.diag` routine, so perhaps `diag` is ok.

Reply:
I think `matrix` is fine (when next to `right`). I can change `diagonal -> diag`, sounds sufficiently descriptive to me.

Reply:
ok, sounds good.